spec/features/user_browses_loan_requests_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.feature "user browses items" do
+  let(:create_item) { LoanRequest.create( ) }
```

Comment:
Do not use parentheses for method calls with no arguments.<br>Space inside parentheses detected.